packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -40,7 +40,9 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
 
     const { _classNames } = this;
 
-    const numPersonasToShow: number = Math.min(personas.length, maxDisplayablePersonas || personas.length);
+    // Add a check to make sure maxDisplayalePersonas is defined to cover the edge case of it being 0.
+    const numPersonasToShow: number = typeof maxDisplayablePersonas !== 'undefined' ?
```

Comment:
Would be slightly better to check `typeof maxDisplayablePersonas === 'number'` instead of checking for undefined